.gitignore

```diff
@@ -31,3 +31,6 @@ dist/
 
 #mvn targets
 presto-admin-test/target
+
+# presto yarn package for product tests
+presto-yarn-package.zip
```

Comment:
If you end up downloading it automatically, it gets this filename:

https://github.com/prestodb/presto-admin/pull/83/files#diff-d7ed4990e37abbc78fb669f18c7cfe1cR116

This follows what we do with the presto-server-rpm.